release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -453,6 +453,14 @@ The egress router CNI plug-in is generally available. The Cluster Network Operat
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/using-an-egress-router-ovn.adoc[Considerations for the use of an egress router pod].
 
+[id="ocp-4-8-control-dns-pod-placement"]
+==== Control DNS pod placement
+
+In {product-title} 4.8, you can use a custom node selector to configure the daemon set for CoreDNS to run or not run on certain nodes.
```

Comment:
Might as well mention that tolerations are configurable too if there isn't a reason not to:
```suggestion
In {product-title} 4.8, you can use a custom node selector and tolerations to configure the daemon set for CoreDNS to run or not run on certain nodes.
```